lib/src/components/buttons/rounded_button.dart

```diff
@@ -61,10 +60,11 @@ class _RoundedButtonState extends State<RoundedButton> {
                     decoration: _isHovered ? TextDecoration.underline : null,
                   ),
             ),
-            const Icon(
-              SFSymbols.chevron_right,
-              size: _iconSize,
+            const Chevron(
+              size: Size.square(_iconSize / 2),
               color: Colors.blueAccent,
+              lineWidth: 2,
+              direction: ChevronDirection.right,
```

Comment:
This one's not used in 1st and 2nd widgets, it will be up to @solid-oleksandrhordiienko to update this with real values.